tests/utils/test_verification_utils.py

```diff
@@ -28,21 +28,35 @@ def test_recover_relative_pose_from_essential_matrix(self):
         self.assertTrue(i2Ri1.equals(i2Ei1.rotation(), 1e-3))
         self.assertTrue(i2Ui1.equals(i2Ei1.direction(), 1e-3))
 
+    def test_recover_relative_pose_from_essential_matrix_none(self):
+        """Test for function to extract relative pose from essential matrix."""
+
+        # simulate correspondences and the essential matrix
+        corr_i1, corr_i2, _ = simulate_two_planes_scene(10, 10)
+
+        i2Ri1, i2Ui1 = verification_utils.recover_relative_pose_from_essential_matrix(
+            None, corr_i1.coordinates, corr_i2.coordinates, Cal3Bundler(), Cal3Bundler()
+        )
+
+        # compare the recovered R and U with the ground truth
+        self.assertIsNone(i2Ri1)
+        self.assertIsNone(i2Ui1)
+
     def test_compute_epipolar_distances(self):
         """Test for epipolar distance computation using 2 sets of points and the essential matrix."""
-        normalized_pts_i1 = np.array([[1.0, 3.5], [-2.0, 2.0]])
-        normalized_pts_i2 = np.array([[2.0, -1.0], [1.0, 0.0]])
+        points_i1 = np.array([[1.0, 3.5], [-2.0, 2.0]])
```

Comment:
can the comment include a little more of the math, for how we get 1.637 and 1.850?

Reply:
Umm I might have to add 2-3 lines.

Reply:
done